tools/flaky_test_monitor/process_results.go

```diff
@@ -218,6 +227,20 @@ func postProcessTestRun(packageResultMap map[string]*PackageResult) {
 		}
 
 		for _, testResults := range packageResult.TestMap {
+			for j, testResult := range testResults {
+				// found nil test - test with no result - due to `go test` bug
```

Comment:
Are you suggesting we change the name `nil test` to something else? I called it that because there's no result for that test even though it ran. Whether it was a parsing issue or something else is irrespective of the fact that the test did not generate a result. Open to other naming suggestions.